partcad/src/partcad/exception.py

```
@@ -10,3 +10,27 @@
 
 class NeedsUpdateException(Exception):
     pass
+
+class PartFactoryError(Exception):
+    """Base exception for all part factory-related errors."""
+    pass
```

Ruff (0.8.2)

Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)